src/Common/Id.php

```diff
@@ -24,5 +24,7 @@
 
 interface Id
 {
-    public function __toString(): string;
+    public function toString(): string;
+
+    public function equals(Id $other): bool;
 }
```

Comment:
I thought that this interface is destinated for entities. I can imagine a situation that some class which uses this interface has `id` but does not need `equals` method (ISP?).
Should be discussed.